<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns='http://www.w3.org/1999/xhtml' xml:lang='en' lang='en'>
<head>
<title>mingw-ilmbase.git - mingw-ilmbase</title>
<meta name='generator' content='cgit v0.10.1'/>
<meta name='robots' content='index, nofollow'/>
<link rel='stylesheet' type='text/css' href='/cgit-data/cgit.css'/>
<link rel='shortcut icon' href='/favicon.ico'/>
<link rel='alternate' title='Atom feed' href='http://pkgs.fedoraproject.org/cgit/mingw-ilmbase.git/atom/ilmbase-2.1.0-win32_threads.patch?h=master' type='application/atom+xml'/>
</head>
<body>
<div id='cgit'><table id='header'>
<tr>
<td class='logo' rowspan='2'><a href='/cgit/'><img src='/cgit-data/cgit.png' alt='cgit logo'/></a></td>
<td class='main'><a href='/cgit/'>index</a> : <a title='mingw-ilmbase.git' href='/cgit/mingw-ilmbase.git/'>mingw-ilmbase.git</a></td><td class='form'><form method='get' action=''>
<select name='h' onchange='this.form.submit();'>
<option value='f19'>f19</option>
<option value='f20'>f20</option>
<option value='f21'>f21</option>
<option value='master' selected='selected'>master</option>
</select> <input type='submit' name='' value='switch'/></form></td></tr>
<tr><td class='sub'>mingw-ilmbase</td><td class='sub right'>Jon Ciesla</td></tr></table>
<table class='tabs'><tr><td>
<a href='/cgit/mingw-ilmbase.git/'>summary</a><a href='/cgit/mingw-ilmbase.git/refs/'>refs</a><a href='/cgit/mingw-ilmbase.git/log/ilmbase-2.1.0-win32_threads.patch'>log</a><a class='active' href='/cgit/mingw-ilmbase.git/tree/ilmbase-2.1.0-win32_threads.patch'>tree</a><a href='/cgit/mingw-ilmbase.git/commit/ilmbase-2.1.0-win32_threads.patch'>commit</a><a href='/cgit/mingw-ilmbase.git/diff/ilmbase-2.1.0-win32_threads.patch'>diff</a><a href='/cgit/mingw-ilmbase.git/stats/ilmbase-2.1.0-win32_threads.patch'>stats</a></td><td class='form'><form class='right' method='get' action='/cgit/mingw-ilmbase.git/log/ilmbase-2.1.0-win32_threads.patch'>
<select name='qt'>
<option value='grep'>log msg</option>
<option value='author'>author</option>
<option value='committer'>committer</option>
<option value='range'>range</option>
</select>
<input class='txt' type='text' size='10' name='q' value=''/>
<input type='submit' value='search'/>
</form>
</td></tr></table>
<div class='path'>path: <a href='/cgit/mingw-ilmbase.git/tree/'>root</a>/<a href='/cgit/mingw-ilmbase.git/tree/ilmbase-2.1.0-win32_threads.patch'>ilmbase-2.1.0-win32_threads.patch</a></div><div class='content'>blob: 182bfa11bf7a7e150e59018d57b1dbc127c31bb3 (<a href='/cgit/mingw-ilmbase.git/plain/ilmbase-2.1.0-win32_threads.patch'>plain</a>)
<table summary='blob content' class='blob'>
<tr><td class='linenumbers'><pre><a id='n1' href='#n1'>1</a>
<a id='n2' href='#n2'>2</a>
<a id='n3' href='#n3'>3</a>
<a id='n4' href='#n4'>4</a>
<a id='n5' href='#n5'>5</a>
<a id='n6' href='#n6'>6</a>
<a id='n7' href='#n7'>7</a>
<a id='n8' href='#n8'>8</a>
<a id='n9' href='#n9'>9</a>
<a id='n10' href='#n10'>10</a>
<a id='n11' href='#n11'>11</a>
<a id='n12' href='#n12'>12</a>
<a id='n13' href='#n13'>13</a>
<a id='n14' href='#n14'>14</a>
<a id='n15' href='#n15'>15</a>
</pre></td>
<td class='lines'><pre><code>diff -rupN ilmbase-2.1.0/IlmThread/Makefile.am ilmbase-2.1.0-new/IlmThread/Makefile.am
--- ilmbase-2.1.0/IlmThread/Makefile.am	2013-06-18 21:51:38.000000000 +0200
+++ ilmbase-2.1.0-new/IlmThread/Makefile.am	2013-12-26 21:25:06.201173876 +0100
@@ -6,9 +6,8 @@ libIlmThread_la_SOURCES = IlmThreadPool.
 			  IlmThreadSemaphore.h IlmThreadMutex.h \
 			  IlmThreadPool.cpp IlmThread.cpp \
 			  IlmThreadSemaphore.cpp IlmThreadMutex.cpp \
-			  IlmThreadPosix.cpp IlmThreadSemaphorePosix.cpp \
-			  IlmThreadSemaphorePosixCompat.cpp \
-			  IlmThreadMutexPosix.cpp
+			  IlmThreadWin32.cpp IlmThreadSemaphoreWin32.cpp \
+			  IlmThreadMutexWin32.cpp
 
 libIlmThread_la_LDFLAGS = -version-info @LIBTOOL_VERSION@ -no-undefined
 if LIB_SUFFIX_EXISTS
</code></pre></td></tr></table>
</div> <!-- class=content -->
<div class='footer'>generated  by cgit v0.10.1 at 2014-08-28 07:12:05 (GMT)</div>
</div> <!-- id=cgit -->
</body>
</html>
